mix.exs

```diff
@@ -68,16 +66,16 @@ defmodule EctoVista.MixProject do
     [
       maintainers: ["Nick Chernyshev"],
       licenses: ["Apache 2.0"],
-      links: %{"GitHub" => "https://github.com/flowerett/ecto_vista"},
+      links: %{"GitHub" => @source_url},
       files: ~w(.formatter.exs mix.exs README.md lib)
     ]
   end
 
   defp docs do
     [
-      main: "EctoVista",
-      extras: ["README.md"],
-      source_url: "https://github.com/flowerett/ecto_vista"
+      main: "readme",
```

Comment:
Hm... not sure if it makes sense to use readme as the main page.
I've copied and updated docs in the main module - `EctoVista` recently, and would like to keep it as a central source of truth.
We can keep only short excerpt in the Readme, just for a quick start.
So... let's keep EctoVista here.

Maybe an additional link (from readme to the main documentation page) will be useful.

Reply:
> Hm... not sure if it makes sense to use readme as the main page.

I remembered because originally the default page `Ecto.Vista` was not found, hence I set it to `readme` page.

> I've copied and updated docs in the main module - `EctoVista` recently, and would like to keep it as a central source of truth.
> We can keep only short excerpt in the Readme, just for a quick start.
> So... let's keep EctoVista here.
> Maybe an additional link (from readme to the main documentation page) will be useful.

I've noticed that there are duplicated content between the module and readme file. Hence, I put in readme but also duplicated to Ecto.Vista module.

If we set the default page to `Ecto.Vista`, the result is as shown. Note that there is no benefits of setting it to default page as this module don't have any extra details like function or callback list. We also lost the details of the badges, which are relevant for a quick overview and have a list of links to navigate to.

![image](https://user-images.githubusercontent.com/134518/99386542-98cfc900-290d-11eb-9999-3686cc88cc36.png)

Using the readme as default page, we've the following.

![image](https://user-images.githubusercontent.com/134518/99387231-97eb6700-290e-11eb-9178-78bd2e5a98eb.png)

Have a look at the recent changes and let me know your preference. Off course, we can also embed selected content from readme to the `Ecto.Vista` module, if you want to set `Ecto.Vista` as source of truth.

Reply:
Not sure if badges are necessary for the documentation, I would rather focus on writing more relevant guides.
Phoenix framework can be a good example here:
https://github.com/phoenixframework/phoenix/tree/master/guides
So maybe we can introduce something like this?

Reply:
> Not sure if badges are necessary for the documentation, I would rather focus on writing more relevant guides.
> Phoenix framework can be a good example here:
> https://github.com/phoenixframework/phoenix/tree/master/guides
> So maybe we can introduce something like this?

Do you foresee this module going to have more docs that cannot fit within the `README.md` file?
If so, then we should put within the `guides` folder. Otherwise, the usage content within the `README.md` should be sufficient enough?

Reply:
I would say it's sufficient at the moment, so let's keep readme for a while.
But answering your first question - yes, I would rather keep readme short with additional link to guides or docs.